packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts

```diff
@@ -735,6 +735,61 @@ describe('Routing', () => {
         .get('/greet/world')
         .expect(200, 'HELLO WORLD');
     });
+
+    it('redirect swagger routes', async () => {
+      const app = new RestApplication();
+      await whenIMakeRequestTo(app)
+        .get('/swagger-ui')
+        .expect(308);
+      await whenIMakeRequestTo(app)
+        .get('/explorer')
+        .expect(308);
+    });
+
+    it('redirect controller routes', async () => {
```

Comment:
I am confused about this test. Are you trying to verify that redirects take precedence over controller methods? Let's capture the intention in the test name please. For example:

```ts
it('gives precedence to redirect routes over controller methods', async () => {
```